src/meta-srv/src/procedure/create_table.rs

```diff
@@ -228,18 +228,14 @@ impl CreateTableProcedure {
         let _ = join_all(joins)
             .await
             .into_iter()
-            .map(|result| {
-                result.map_err(|err| {
-                    error::RetryLaterSnafu {
-                        reason: format!(
-                            "Failed to execute create table on datanode, source: {}",
-                            err
-                        ),
-                    }
-                    .build()
-                })
-            })
-            .collect::<Result<Vec<_>>>()?;
+            .map(|e| e.context(error::JoinSnafu).flatten())
+            .collect::<Result<Vec<_>>>()
+            .map_err(|err| {
+                error::RetryLaterSnafu {
```

Comment:
Is creating a table in DN an idempotent operation? I ask because it seems that if there are any partial failures, the entire process will be retried.

Reply:
Yep, Let me change it to retry on network errors.